validator/validator.go

```diff
@@ -39,12 +40,15 @@ func (val *Validator) Number() int                 { return val.data.Number }
 func (val *Validator) Sequence() int               { return val.data.Sequence }
 func (val *Validator) Stake() int64                { return val.data.Stake }
 func (val *Validator) BondingHeight() int          { return val.data.BondingHeight }
+func (val *Validator) UnbondingHeight() int        { return val.data.UnbondingHeight }
 func (val *Validator) LastJoinedHeight() int       { return val.data.LastJoinedHeight }
 
 func (val Validator) Power() int64 {
 	// Only bootstrap validators at genesis block has no stake
 	if val.data.Stake == 0 {
 		return 1
+	} else if val.data.UnbondingHeight > 0 { //if the validator requested to unbound ignore stake
```

Comment:
We don't need to unbond bootstrap validators because they don't have any stake. So this check is redundant.